src/desktop/DesktopMain.js

```diff
@@ -26,7 +26,7 @@ import {ElectronNotificationFactory} from "./NotificatonFactory"
 import {KeytarSecretStorage} from "./sse/SecretStorage"
```

Comment:
I think it's much better than before. It still has init(), startupInstance() and main(). We still handle argv in two places outside of handleArgv() and `wasAutolaunched` is inside components? If think if you touch this we should fix at least some of these